src/components/DuplicatesPage/DuplicatesTable/DuplicatesTable.js

```diff
@@ -0,0 +1,88 @@
+import * as React from "react";
+import DuplicateTableRow from "./DuplicateTableRow/DuplicateTableRow";
+import {arrayOf, func, number, object} from "prop-types";
+import "./duplicatesTable.less";
+import {EXTENDED_DUPLICATES_TABLE_HEADERS} from "../../../constants/duplicatesHeaders";
+import {areAllFieldsSelected, buildMergedItem, getDuplicatesIds} from "../../../utils/duplicates";
+import Table from "../../Common/Table/Table";
+import SelectableTableCreator from "../../Composable/SelectableTableCreator/SelectableTableCreator";
+import ButtonsBar from "../../Common/ButtonsBar/ButtonsBar";
+
```

Comment:
If you are importing entire module ---> go for import * as duplicates from "../../../utils/duplicates" which has better semantics. If you are using a library such as lodash or React or prop types --> do partial importing for performance.